MIDAS Journals.js

```diff
@@ -0,0 +1,122 @@
+   var authorXPath = doc.evaluate('//*[@id="publication"]/div[@class="authors"]', doc, nsResolver, XPathResult.ANY_TYPE, null); 
+   var urlXPath = doc.evaluate('//*[@id="publication"]/table/tbody/tr/td/a', doc, nsResolver, XPathResult.ANY_TYPE, null); 
+   var journalXPath = doc.evaluate('//*[@id="publication"]/div[@class="journal"]/a[1]', doc, nsResolver, XPathResult.ANY_TYPE, null); 
+   var issueXPath = doc.evaluate('//*[@id="publication"]/div[@class="journal"]/a[2]', doc, nsResolver, XPathResult.ANY_TYPE, null); 
+   var submittedXPath = doc.evaluate('//*[@id="publication"]/div[@class="submittedby"]', doc, nsResolver, XPathResult.ANY_TYPE, null); 
+   var abstractXPath = doc.evaluate('//*[@id="publication"]/div[@class="abstract"]', doc, nsResolver, XPathResult.ANY_TYPE, null); 
+   var downloadAllXPath = doc.evaluate('//a[contains(text(),"Download All")]/@href', doc, nsResolver, XPathResult.ANY_TYPE, null); 
+   var pdfXPath1 = doc.evaluate('//a[contains(text(),"Download Paper")]/@href', doc, nsResolver, XPathResult.ANY_TYPE, null); 
+   //var pdfXPath2 = doc.evaluate('//a[contains(text(),".pdf")]/@href', doc, nsResolver, XPathResult.ANY_TYPE, null); 
+ 
+ 
+   pdfhref=pdfXPath1.iterateNext();
+   //if(!pdfhref) {
+   //	 pdfhref=pdfXPath2.iterateNext();
+   //}
+   if(pdfhref) {
+	  tmp=pdfhref.baseURI.split('/');
+	  pdflink=tmp[0]+'//' +tmp[2]+pdfhref.value;
+	  //Zotero.debug(pdflink);
+   }
+   datematch=new RegExp("on\\w+([0-9]+)\\-([0-9]+)\\-([0-9]+)");
+   datematch=new RegExp("on +([0-9]+)\\-([0-9]+)\\-([0-9]+)");
+   submittedString=submittedXPath.iterateNext().textContent;
+   dateparse=datematch.exec(submittedString);
+  
+   authors=authorXPath.iterateNext().textContent.split(/, */);
+   var newItem = new Zotero.Item("journalArticle");
+   newItem.title = titleXPath.iterateNext().textContent;
+   newItem.url = urlXPath.iterateNext().textContent; 
+   newItem.publicationTitle = journalXPath.iterateNext().textContent;
+   newItem.seriesTitle = issueXPath.iterateNext().textContent;
+   newItem.abstractNote = abstractXPath.iterateNext().textContent;
+   newItem.date = dateparse[3]+"-"+dateparse[1]+"-"+ dateparse[2];
+   splitDownloadPath = downloadAllXPath.iterateNext().value.split('/');
+   version=splitDownloadPath[splitDownloadPath.length-1]
+   newItem.extra="Revision: " + version
+   newItem.issue=splitDownloadPath[splitDownloadPath.length-2]
+
+   var a=0;
+   while (a<authors.length) {
+   	   splitname=authors[a].split(' ');
+   	   a=a+1;
+   	   //Zotero.debug("splitname");
+   	   //Zotero.debug(splitname);
+   	   var z=1;
+   	   name="";
+   	   while (z<splitname.length) {
+   	   	  name=name+splitname[z]+" ";
+   	   	  z=z+1;
+   	   }
+   	   name=name+splitname[0];
+   	   newItem.creators.push(Zotero.Utilities.cleanAuthor(name,"author"));
+   }
+   //author
+   //date
+   //pdf	
+   if(pdfhref) {
+   		newItem.attachments.push({
+		   title: newItem.publicationTitle+" "+newItem.issue + " v" + version + " PDF",
```

Comment:
Simply "Full Text PDF" is sufficient. The specifics are available in the metadata